x/claim/types/params.go

```diff
@@ -1,11 +1,81 @@
 package types
 
 import (
+	"fmt"
 	"time"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
+// Parameter store keys
 var (
-	DefaultClaimDenom         = "uosmo"
-	DefaultDurationUntilDecay = time.Hour
-	DefaultDurationOfDecay    = time.Hour * 5
+	KeyClaimDenom         = []byte("ClaimDenom")
+	KeyDurationUntilDecay = []byte("DurationUntilDecay")
+	KeyDurationOfDecay    = []byte("DurationOfDecay")
 )
```

Comment:
I think we should undo this. I don't think details of the airdrop should be on-chain parameters. If these were changed, it should be forced to go through a state machine upgrade, due to how intrusive these are.

Reply:
true thought we might need to change them to proper param form at some point, but coming back to it, were not using them as params so should be fine to go back to the original form

Reply:
Not sure why convo was marked as resolved. These should be deleted / weren't undone?

Reply:
Hrmm, these keys are still showing up in files changed. I'll check out the PR / make a commit tho